config/routes.rb

```diff
@@ -5,6 +5,9 @@
   get "/applications/:id", to: "applications#show"
   post "/applications", to: "applications#create"
 
+
+  post "/applications/:id/pet_applications", to: "pet_applications#create"
```

Comment:
I've personally never made a route that doesn't have a show page, for some reason they seem synonymous, but this a good reminder that they are not!